### ISSUE TITLE
French translation

### ISSUE THREAD

**Issue by Calinou** [designbyjake/svgontheweb#10]
Is there anyone working on a French translation? If not, this is something I can tackle.

**Comment by designbyjake** [comment]
@Calinou There is not, that would be great!

**Comment by Calinou** [comment]
I began working on a French translation, it is about 40% done now.

**Comment by designbyjake** [comment]
@Calinou Have you made any more progress on this? (Just checking in, no rush)

**Comment by Calinou** [comment]
Yes, it's about 80% done now, a few sentences still need to be translated.

**Comment by jakegiltsoff** [comment]
@Calinou Hello, just wanted to check if you have an update on this?

**Comment by Calinou** [comment]
I didn't work on it for a while, but I still have it around locally.
